pytest_order/sorter.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 import sys
 from warnings import warn
-from collections import OrderedDict
-from typing import Optional, List, Dict, Tuple
+from contextlib import suppress
+from typing import Optional, List, Dict, Tuple, OrderedDict, cast
```

Comment:
`OrderedDict` is not available in 3.6, I think

Reply:
Yeah, there is 3 ways of getting out of that here:

- Drop 3.6 support, which is going to reach EOL in ~8 months, according to the schedule. Simple, effective, and lets you avoid having to drop it later. May not be something you're after here though.
- Omit the typings - each line specifying OrderedDict would get a `# type: ignore` comment. Kinda messy, specifically since this would omit the typings for all versions, beyond 3.6 too.
- Stub it - It should be possible to dynamically define an `OrderedDict` generic typing class, probably by following what the docs say, and inheriting `MutableMapping` and `collections.OrderedDict`. Could be gated behind a Python version check, although the docs say it was added in `3.7.2`, so maybe a try-except with an `ImportError` handler defining it instead, would be better. The type checker may, but shouldn't be, confused by a definition like that. This is probably the second-best solution, after the first one presented.

The choice is up to you, of course. Which one would you prefer to go with?

Reply:
I don't want to drop support for 3.6 just because of a typing issue. I dropped support for 2.7 and 3.5 only just now, because I know that they have been out there a lot still (even if I didn't like not to be able to use typing, f-strings or enums). I think I'll drop support for 3.6 a few months after EOL, longer if there are Linux distributions out there that still use it.
If you want to go with stubbing it - go ahead! I find that generally a bit messy, I'm also ok with ignoring it in this case, but I leave that up to you.